jsonschema/tests/test_validators.py

```diff
@@ -191,6 +191,26 @@ def test_invalid_format_default_message(self):
         self.assertIn(repr("thing"), message)
         self.assertIn("is not a", message)
 
+    def test_additionalProperties_false_patternProperties(self):
+        schema = {u"type": u"object",
+                  u"additionalProperties": False,
+                  u"patternProperties": {
+                      u"$abc^": {u"type": u"string"},
```

Comment:
Nitpick: can you change these to slightly more realistic regexes, e.g. `^abc$` :)?